ci/scripts/audit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-authorisation
+  make audit
```

Comment:
`Makefile` is missing `audit` and `build`

Reply:
Pushed changes thanks :-)